src/main/kotlin/com/exactpro/th2/rptdataprovider/Extensions.kt

```diff
@@ -58,6 +61,48 @@ suspend fun <T> logTime(methodName: String, lambda: suspend () -> T): T? {
     }
 }
 
+
+suspend fun Writer.eventWrite(event: SseEvent) {
+    withContext(Dispatchers.Default) {
+        if (event.event != null) {
+            write("event: ${event.event}\n")
+        }
+
+        for (dataLine in event.data.lines()) {
+            write("data: $dataLine\n")
+        }
+
+        if (event.id != null) {
+            write("id: ${event.id}\n")
+        }
+        write("\n")
+        flush()
+    }
+}
+
+suspend fun Writer.asyncClose() {
+    withContext(Dispatchers.IO) {
+        close()
+    }
+}
+
+fun Instant.min(other: Instant): Instant {
```

Comment:
There is a function in the standard library called minOf. If you want the same thing, it's better to rename this and change the signature to avoid confusion.